ros2service/ros2service/verb/echo.py

```diff
@@ -0,0 +1,236 @@
+# Copyright 2022 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import sys
+from typing import Optional
+from typing import TypeVar
+
+from collections import OrderedDict
+
+import rclpy
+from rclpy.node import Node
+from rclpy.qos import QoSPresetProfiles
+
+from ros2cli.node.strategy import NodeStrategy
+from ros2topic.api import unsigned_int
+from rcl_interfaces.msg import ServiceEventType
+
+from ros2service.api import ServiceNameCompleter, get_service_class
+from ros2service.api import ServiceTypeCompleter
+from ros2service.verb import VerbExtension
+
+from rosidl_runtime_py import message_to_csv, message_to_ordereddict
+from rosidl_runtime_py.utilities import get_message, get_service
+
+import yaml
+
+DEFAULT_TRUNCATE_LENGTH = 128
+MsgType = TypeVar('MsgType')
+
+
+def represent_ordereddict(dumper, data):
+    items = []
+    for k, v in data.items():
+        items.append((dumper.represent_data(k), dumper.represent_data(v)))
+    return yaml.nodes.MappingNode(u'tag:yaml.org,2002:map', items)
+
+
+class EchoVerb(VerbExtension):
+    """Echo a service."""
```

Comment:
Could you expand this docstring to provide a short explanation and an example ? It would be nice if we could write a simple test / self contained example for this feature.

Reply:
Yes, we should add a test to [test_cli.py](https://github.com/ros2/ros2cli/blob/rolling/ros2service/test/test_cli.py).